<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>monitor_autoadmin_basedialog</class>
 <widget class="QDialog" name="monitor_autoadmin_basedialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>516</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Schedule: Auto Admin tasks</string>
  </property>
  <property name="windowIcon">
   <iconset resource="gex.qrc">
    <normaloff>:/gex/icons/gex_application_48x48.png</normaloff>:/gex/icons/gex_application_48x48.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="pixmapLabel1_2">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="pixmap">
        <pixmap resource="../../gex.qrc">:/gex/icons/calendar_icon.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="TextLabel1">
       <property name="font">
        <font>
         <family>Ms Sans Serif</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Task: Auto Admin</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox_OkCancel">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="TextLabel2_2_3">
       <property name="text">
        <string>Title:</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="LineEditTitle">
       <property name="toolTip">
        <string>Enter task title!</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget2">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tabAdmin">
      <attribute name="title">
       <string>Admin</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="TextLabel2_2_2_2_4_2_2_2">
         <property name="text">
          <string>Specify admin &amp; cleaning tasks to take place automatically every day...</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>542</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="TextLabelStopWindow_2">
           <property name="text">
            <string>Log file contents:</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="comboBoxLogContent">
           <property name="toolTip">
            <string>Information to report in the History tab (log file)</string>
           </property>
           <item>
            <property name="text">
             <string>Exceptions only (Errors and Alarms)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>All tasks details</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="TextLabelStopWindow">
           <property name="text">
            <string>Daily Auto-Admin starts at:</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QTimeEdit" name="timeEditStart">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="TextLabel2_2_2_2_4_4">
           <property name="text">
            <string>Erase reports on server:</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="comboBoxEraseReports">
           <property name="toolTip">
            <string>Automatic cleanup of old reports on the server...</string>
           </property>
           <item>
            <property name="text">
             <string>If older than 1 day</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>If older than 2 days</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>If older than 3 days</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>If older than 4 days</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>If older than 5 days</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>If older than 6 days</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>If older than 1 week</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>If older than 2 weeks</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>If older than 3 weeks</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>If older than 1 Month</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>If older than 2 Months</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>If older than 3 Months</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>If older than 4 Months</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>If older than 5 Months</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>If older than 6 Months</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Never</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="TextLabel2_2_2_2_4_3">
           <property name="text">
            <string>Email from:</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="lineEditEmailFrom">
           <property name="toolTip">
            <string>Enter 'From' email address</string>
           </property>
           <property name="text">
            <string>no-reply@mentor.com</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="TextLabel2_2_2_2_4_2">
           <property name="text">
            <string>Sys. Admin Email:</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLineEdit" name="lineEditEmailList">
             <property name="toolTip">
              <string>Enter email addresses to notify (comma separated) on admin issues</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonMailingList">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>26</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>26</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Pick a mailing list file</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../../gex.qrc">
               <normaloff>:/gex/icons/email.png</normaloff>:/gex/icons/email.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="5" column="1">
          <widget class="QCheckBox" name="checkBoxSendLogs">
           <property name="text">
            <string>Send daily insertion logs</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabAlarmShells">
      <attribute name="title">
       <string>Notifications Shell</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>583</width>
            <height>374</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4"/>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>LineEditTitle</tabstop>
  <tabstop>lineEditEmailList</tabstop>
 </tabstops>
 <resources>
  <include location="../../gex.qrc"/>
 </resources>
 <connections/>
</ui>
